app/controllers/application_controller.rb

```diff
@@ -9,6 +9,10 @@ def authorize_admin!
     end
   end
 
+  def after_sign_in_path_for(resource)
```

Comment:
Unused method argument - `resource`. If it's necessary, use `_` or `_resource` as an argument name to indicate that it won't be used. You can also write as `after_sign_in_path_for(*)` if you want the method to accept any arguments but don't care about them.